### PR TITLE
Make `Acknowledgment` spelling consistent

### PR DESCRIPTION
**Pull request by Jarema**
Found a typo in the Consumer state. `Acknowledgment` -> `Acknowledgement`.

### CODE REVIEW

File: cli/consumer_command.go [hunk 1]
```diff
@@ -658,9 +658,9 @@ func (c *consumerCmd) showInfo(config api.ConsumerConfig, state api.ConsumerInfo
 
 	if config.AckPolicy != api.AckNone {
 		if state.AckFloor.Last == nil {
-			fmt.Printf("     Acknowledgment floor: Consumer sequence: %s Stream sequence: %s\n", humanize.Comma(int64(state.AckFloor.Consumer)), humanize.Comma(int64(state.AckFloor.Stream)))
+			fmt.Printf("     Acknowledgement floor: Consumer sequence: %s Stream sequence: %s\n", humanize.Comma(int64(state.AckFloor.Consumer)), humanize.Comma(int64(state.AckFloor.Stream)))
```
Comment (ripienaar) [inline cli/consumer_command.go:661]: In american it's Acknowledgment and in british its Acknowledgement.  Both are corect but we tend to use american english here

Comment (Jarema) [reply cli/consumer_command.go:661]: Sure, but checking the codebase, everywhere else British version is used.
Should not call it "typo" though ;).
Those are only two uses of american ones for that word.

Examples:
```
	if c.maxAckPending == -1 && cfg.AckPolicy != api.AckNone {
		err = askOne(&survey.Input{
			Message: "Maximum Acknowledgements Pending",
			Default: "0",
			Help:    "The maximum number of messages without acknowledgement that can be outstanding, once this limit is reached message delivery will be suspended. Settable using --max-pending.",
		}, &c.maxAckPending)
		fisk.FatalIfError(err, "could not ask for maximum outstanding acknowledgements")
	}
```

Comment (ripienaar) [reply cli/consumer_command.go:661]: Think we should make those American then :)

Comment (Jarema) [reply cli/consumer_command.go:661]: Agree, will "reverse" this PR approach :)

**Review by ripienaar — APPROVED** [review]
LGTM

**Pull request merged**